### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by phecho**

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Probably should be "Invalid email/username or password combination"

**Comment by phecho** [comment]
I don't think it is necessary. User should know what the username stands for.
just like github:
image

what about  "Invalid login or password combination" ?

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
                     <div class="form-group">
                         <label class="sr-only">{{ trans('forms.login.login') }}</label>
-                        <input autocomplete="off" class="form-control login-input" placeholder="{{ trans('forms.login.login') }}" required="required" name="login" type="text" autofocus>
+                        <input autocomplete="off" class="form-control login-input" placeholder="{{ trans('forms.login.login') }}" required="required" name="login" type="text" value="{{Binput::old('login')}}" autofocus>
```
Comment (GrahamCampbell) [inline resources/views/auth/login.blade.php]: cs

**Comment by jbrooksuk** [comment]
Ok, I can go with that :)

File: resources/views/auth/login.blade.php (hunk 1 above)
Comment (jbrooksuk) [inline resources/views/auth/login.blade.php]: Fix and we can merge :)

**Comment by jbrooksuk** [comment]
Cheers!

**Comment by GrahamCampbell** [comment]
@jbrooksuk The language files are now out of sync.

**Comment by jbrooksuk** [comment]
I need to upload them to CrowdIn.

**Pull request merged**